sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Supported `backends` for `LoadBalancingRule`.
 - Supported `withSslPolicy` for `ApplicationGateway`.
+- Now `ApplicationGateway` can be created with `BASIC` sku and tier if not specified.
```

Comment:
Should we move to breaking changes for notification for behavior change.

```
No new V1 deployments: August 28, 2024 - V1 creation is stopped completely for all customers 28 August 2024 onwards.
```

Reply:
Sure.